lib-sql/tables.sql

```diff
@@ -244,6 +244,22 @@ CREATE UNIQUE INDEX idx_postcode_id ON location_postcode USING BTREE (place_id)
 CREATE INDEX idx_postcode_geometry ON location_postcode USING GIST (geometry) {{db.tablespace.address_index}};
 GRANT SELECT ON location_postcode TO "{{config.DATABASE_WEBUSER}}" ;
 
+-- Table to store location of entrance nodes
+DROP TABLE IF EXISTS place_entrance;
+CREATE TABLE place_entrance (
+  place_id BIGINT NOT NULL,
+  entrances JSONB NOT NULL
```

Comment:
I wonder if at this point it makes sense to make this table more structured, and replace the EntranceDetails type with the model for this table. Something like:
```sql
  place_id BIGINT NOT NULL,
  osm_id BIGINT NOT NULL,
  type TEXT NOT NULL,
  location GEOMETRY(Point, 4326) NOT NULL,
  extratags JSONB

Reply:
Possibly. It would very likely be more efficient to save on the PostgreSQL side. By moving the code that reads the table to the extra data section, you very likely no longer need the SQLite json function hell that prompted to switch to the compact table in the first place.